storage/interceptors/smaap_service_binding_interceptor.go

```diff
@@ -112,6 +112,12 @@ func (i *ServiceBindingInterceptor) AroundTxCreate(f storage.InterceptCreateArou
 			return nil, fmt.Errorf("operation missing from context")
 		}
 
+		if instance.ReferencedInstanceID != "" {
```

Comment:
how about AroundTxUpdate - should not we skip going to the broker for reference binding updates?

Reply:
Hey, 
As far as I know, there is no AroundTxUpdate for bindings.
PMed you about it, waiting for a response before resolving.
Thanks!

Reply:
Done as no need any change, resolving.